hbp_validation_framework/__init__.py

```diff
@@ -212,9 +279,10 @@ class TestLibrary(BaseClient):
     password : string, optional
         Your HBP Collaboratory password; advisable to not enter as plaintext.
         If left empty, you would be prompted for password at run time (safer).
-    url : string, optional
```

Comment:
Same comment as above. We need to keep `url` field.

Reply:
Resolved by introducing `environment` in d5ba8248ef8cd61e8f29f53b75d65e78ee6358a7 and planned feature in https://github.com/apdavison/hbp-validation-client/issues/15